mlrun/feature_store/api.py

```diff
@@ -405,7 +408,12 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "feature set and source must be specified"
         )
-
+    entities_keys = [e.name for e in featureset.spec.entities]
+    for step_name, step_value in featureset.graph.steps.items():
+        if step_name == get_name(name=None, class_name=DropFeatures):
+            dropped_features = step_value.class_args.get("features")
```

Comment:
We don't want to get `None` if this is missing somehow. We could check for that, but I think this change will suffice:
```suggestion
            dropped_features = step_value.class_args["features"]
```

Reply:
important comment! ty!